Makefile

```diff
@@ -1,34 +1,121 @@
-OPERATOR_SDK_VERSION ?= v0.18.2
-IMAGE_REGISTRY ?= quay.io/oyahud
-OPERATOR_IMAGE ?= kubevirt-ssp-operator
-IMAGE_TAG ?= latest
-OPERATOR_BUILD_ARGS ?= "-v -mod=vendor"
+# Current Operator version
+VERSION ?= 0.0.1
+# Default bundle image tag
+BUNDLE_IMG ?= controller-bundle:$(VERSION)
+# Options for 'bundle-build'
+ifneq ($(origin CHANNELS), undefined)
+BUNDLE_CHANNELS := --channels=$(CHANNELS)
+endif
+ifneq ($(origin DEFAULT_CHANNEL), undefined)
+BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
+endif
+BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 
-operator-sdk:
-	curl -JL https://github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_VERSION)/operator-sdk-$(OPERATOR_SDK_VERSION)-x86_64-linux-gnu -o operator-sdk
```

Comment:
I don't see any replacement of this, how do we ensure that operator-sdk version is pinned to the exact

Reply:
`operator-sdk` binary is not needed to build, it was only used to generate the repo, add API and controller code. The make targets use `kustomize` and `controller-gen`. Both of them have fixed versions in the makefile.

We could mention the version in a readme or documentation.